js/languages/en-US.json

```diff
@@ -806,6 +806,7 @@
   },
```

Comment:
@morebrownies would you mind testing this? It's not happening for Josh, but consistently happens for me.

In it's simplest form you:

```
- add coupon for 50% off
- remove the above coupon
- try adding it again.
```

For me it fails with an error that the coupon amount cannot exceed 100% of the price, which is incorrect, because the coupon is only 50% of the price. It's as if it's still considering the removed 50% coupon in the calculation.